tensorflow/python/training/initial_learning_rate_decay.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
I decided to do so because of the confusion you mentioned with using
initial learning rate versus feeding current learning rate into the
function.  I can change it back if you prefer.

On Mon, Mar 28, 2016, 12:51 PM Vijay Vasudevan <notifications@github.com>
wrote:

> In tensorflow/python/training/initial_learning_rate_decay.py
> <https://github.com/tensorflow/tensorflow/pull/1512#discussion_r57622442>:
>
> > @@ -0,0 +1,190 @@
> > +# Copyright 2015 Google Inc. All Rights Reserved.
>
> is there any reason why you split this up into a separate file ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/1512/files/8b94a176675164bf99f6d92e3bace6b5fa5b5bc3#r57622442>
>